addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/handler/DeviceThingHandler.java

```diff
@@ -339,11 +341,19 @@ public void onGroupWrite(AbstractKNXClient client, IndividualAddress source, Gro
                         if (type != null) {
                             OutboundSpec commandSpec = selector.getCommandSpec(configuration, typeHelper, type);
                             if (commandSpec != null) {
-                                GroupAddress mainGa = getKNXChannelMainGA(channel);
-                                // only process if value has changed
-                                if (getRespondingSpecValue(mainGa) != commandSpec.getType()) {
-                                    // if destination is mainGA AND changed value, for mainGA expose next expected
-                                    // GroupValueWrite from openHAB to KNX
+                                // Only process Control Channel, if it is not blocked after a GroupValueResponse was
+                                // sent to KNX by openHAB (set from: onGroupRead "Send REFRESH...").
+                                if (!groupAddressesWriteBlockedOnce.remove(commandSpec.getGroupAddress())) {
+                                    GroupAddress mainGa = getKNXChannelMainGA(channel);
+                                    // Only if destination is mainGA AND changed value, for mainGA expose next expected
+                                    // GroupValueWrite from openHAB to KNX. This prevents echos from openHAB to KNX for
+                                    // GroupValueWrite telegrams sent from KNX to a Control Channel. To send mainGA if
+                                    // changed, simulates openHAB to handle this GA like a Hardware-DTP where T-flag
+                                    // (german:Ü) is set.
+                                    // if (getRespondingSpecValue(mainGa) == commandSpec.getType() &&
```

Comment:
Full replace, yes.
This code is documentation for f-Flag?! Delete and a new force Update?

Reply:
You mean T-flag?
As long as we do not support this option and thus have code commented out here, I would simply remove it.

Reply:
Yes meant T-flag: 
```

T-flag (german:Ü) - set for every GroupAddress in KNX binding
The device will for this object transmit any updated object value, i.e. it will send a GroupValueWrite telegram to the bus. For a push button object this e.g. means that a rocker representing this object was manipulated.

```